manual/manual/foreword.etex

```diff
@@ -68,14 +68,12 @@ The OCaml documentation and user's manual is licensed under a
 
 \begin{latexonly}
 The complete OCaml distribution can be accessed via the Web
-sites \url{http://www.ocaml.org/} and \url{http://caml.inria.fr/}.
-The former Web site contains a lot of additional information on OCaml.
+site \url{https://ocaml.org/}.  This site contains a lot of additional
+information on OCaml.
 \end{latexonly}
 
 \begin{htmlonly}
 The complete OCaml distribution can be accessed via the
-\href{http://www.ocaml.org/}{community Caml Web site} and the
-\href{http://caml.inria.fr/}{older Caml Web site}.
-The \href{http://www.ocaml.org/}{community Caml Web site}
-contains a lot of additional information on OCaml.
+\href{https://ocaml.org/}{ocaml.org Web site}.
```

Comment:
It looks odd to capitalise 'Web' here/

Reply:
My age shows :-)  

"A few editorially conservative publications still use the two-word Web site, but this relic of the 1990s has fallen out of favor throughout the English-speaking world. The one-word, uncapitalized website now prevails by an overwhelming margin.  Exceptions are easily found, however, especially in American sources, where Web site (or web site, without the capital w) appears about once for every six instances of website. This is likely due to the influence of the New York Times, which is notoriously conservative with tech terms. The Times still uses Web site, and many American publications follow suit."

I switched to "website".

Reply:
Indeed, I wouldn't want to be called a "relic of the 1990s". Ouch!

Reply:
It's the expression "Web site" that is a relic of the 1990s, not me personally.  Athough, to some extent...

Reply:
Tsk tsk, the New York Times appears to have been lax in the past! "Placing a link from your website to The New York Times on the Web means that <...>" https://archive.nytimes.com/www.nytimes.com/info/help/linking.html

That was a nice ride back through memory lane though, back when the NYT was browsable with Lynx :-)